### ISSUE TITLE
Editorconfig support

### ISSUE THREAD

**Issue by simonedelmann** [blitz-js/blitz#455]
### What do you want and why?

It would be great to include `.editorconfig` support out of the box. (More details about .editorconfig files [here](https://editorconfig.org).)

### Possible implementation(s)

.editorconfig
```
root = true

[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
indent_style = space
indent_size = 2
trim_trailing_whitespace = true

[*.md]
trim_trailing_whitespace = false
```

### Additional context

- It seems like there is support for `.editorconfig` files within prettier. (https://prettier.io/docs/en/api.html) Should be possible to link them!

**Comment by eliasjohansson** [comment]
Thank you for the request! If it isn't already supported I personally agree that it should be.

**Comment by simonedelmann** [comment]
I did a PR for this! It turned out, linking prettier to editorconfig was super easy.

**Comment by simonedelmann** [comment]
Closed, see #460 for discussion.
